src/options.rs

```diff
@@ -286,20 +314,129 @@ impl<Mode> OptionsBuilder<Mode> where
                 self.mode.parse_builder_name(lit)
             },
             _ => {
-                panic!("Unknown option `{:?}`", ident)
+                panic!("Unknown option `{}`.", ident.as_ref())
+            }
+        }
+    }
+
+    /// e.g `setter(skip)` in `#[builder(setter(skip))]`
+    #[allow(non_snake_case)]
+    fn parse_builder_options_list(
+        &mut self,
+        ident: &syn::Ident,
+        nested: &[syn::NestedMetaItem]
```

Comment:
FYI, you can use trailing commas in param lists to prevent needless diff churn in the future